### PR TITLE
[5082] Add complete? to trainee report

### PR DESCRIPTION
**Pull request by darokel**
### Context

https://trello.com/c/mlYZrxEE/5082-add-to-trainee-exports-if-trainee-is-incomplete-complete

Adds a new column for `is_complete` to the trainee export.

### Changes proposed in this pull request

- Expose `submission_ready` from the trainee to the report class

### Guidance to review

You can check this with the example data in QA. Make sure to use sweet old Annie Bell

https://register-pr-3532.test.teacherservices.cloud/reports/performance-profiles

### CODE REVIEW

**Comment by darokel** [comment]
[Add to trainee exports if trainee is incomplete / complete](https://trello.com/c/mlYZrxEE/5082-add-to-trainee-exports-if-trainee-is-incomplete-complete)

File: app/models/reports/trainees_report.rb [hunk 1]
```diff
@@ -117,6 +118,7 @@ def csv_row(trainee)
       trainee_report = Reports::TraineeReport.new(trainee)
       [
         trainee_report.register_id,
+        trainee_report.complete?,
```
[the remark this answers is not in the document]
Comment (darokel) [reply app/models/reports/trainees_report.rb:121]: Rubocop says no

[the remark this answers is not in the document]
Comment (defong) [reply app/models/reports/trainees_report.rb:121]: https://github.com/DFE-Digital/register-trainee-teachers/blob/c8edbaa06e47df7fa3e15859741f5ab350a97815/app/services/exports/report_csv_enumerator_service.rb#L45-L50

Alias method then please...

[the remark this answers is not in the document]
Comment (darokel) [reply app/models/reports/trainees_report.rb:121]: just renamed the header https://github.com/DFE-Digital/register-trainee-teachers/pull/3532/commits/c293defe1c3e6eeba3e0bc46e2c3b54229022b3a

File: app/models/reports/trainee_report.rb [hunk 2]
```diff
@@ -365,6 +365,10 @@ def submitted_for_trn_at
       trainee.submitted_for_trn_at&.iso8601
     end
 
+    def complete?
+      trainee.submission_ready?
+    end
+
```
[the remark this answers is not in the document]
Comment (darokel) [reply app/models/reports/trainee_report.rb]: https://github.com/DFE-Digital/register-trainee-teachers/pull/3532/commits/c293defe1c3e6eeba3e0bc46e2c3b54229022b3a

**Review by defong — APPROVED** [review]
You the best :1st_place_medal:

**Pull request merged**